client2/pages/index.tsx

```diff
@@ -1,7 +1,24 @@
 import Head from 'next/head';
-import SearchRecipes from '../components/searchRecipes/SearchRecipes';
+import { ReactElement, useState } from 'react';
+import PageLayout from '../components/layouts/pageLayout/PageLayout';
+import RecommendedRecipes from '../components/recommendedRecipes/RecommendedRecipes';
+import { backendService } from '../services/backend';
+import { Recipe } from '../types';
+import type { NextPageWithLayout } from './page';
+
+const Home: NextPageWithLayout = () => {
+  const [recommendedRecipe, setRecommendedRecipe] = useState<Recipe[]>([]);
+
+  const returnRecommendedRecipes = async () => {
+    const data = await backendService.getRecommenedRecipes();
+    setRecommendedRecipe(data);
+    if (!data) {
+      return <p>Page is loading...</p>;
+    }
+  };
+
+  returnRecommendedRecipes();
```

Comment:
change the verb here
```suggestion
  getRecommendedRecipes();
```

Reply:
Done